README.md

```diff
@@ -130,6 +131,9 @@ There isn't a current roadmap for this plugin, but it is in active development.
 
 If you have a feature request, please add it as an [issue](https://github.com/intelsdi-x/snap-plugin-publisher-graphite/issues/new) and/or submit a [pull request](https://github.com/intelsdi-x/snap-plugin-publisher-graphite/pulls).
 
+### Known issues
```

Comment:
Is it necessary to have this `Known Issues` section? Isn't that what the Issues section of Github is for. 
If we keep it, `issues` should be capitalized to match the rest of the casing of headers in this document.